eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -96,7 +99,7 @@ object Helpers {
   /**
    * Called by the fundee
    */
-  def validateParamsFundee(nodeParams: NodeParams, initFeatures: Features, channelFeatures: ChannelFeatures, open: OpenChannel, remoteNodeId: PublicKey): Either[ChannelException, Option[ByteVector]] = {
+  def validateParamsFundee(nodeParams: NodeParams, initFeatures: Features, channelType: SupportedChannelType, open: OpenChannel, remoteNodeId: PublicKey, remoteFeatures: Features): Either[ChannelException, (ChannelFeatures, Option[ByteVector])] = {
```

Comment:
Nit (or `localInitFeatures`/`remoteInitFeatures`?)

```suggestion
  def validateParamsFundee(nodeParams: NodeParams, channelType: SupportedChannelType, localFeatures: Features, open: OpenChannel, remoteNodeId: PublicKey, remoteFeatures: Features): Either[ChannelException, (ChannelFeatures, Option[ByteVector])] = {
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1867/commits/439643cbf3f8215cd913f072021d5259903fa5c8